src/prefect/executors/dask.py

```diff
@@ -238,6 +238,13 @@ def start(self) -> Iterator[None]:
         finally:
             self.client = None
 
+    async def _on_scheduler_message(self, op: str, message: dict) -> None:
```

Comment:
If we intend this to be overridden by users, should we remove the private `_` marking? Can you add a docstring to `on_scheduler_message`?

Reply:
Sure! Will do 👌

Reply:
Done! I changed it to be `on_worker_status_changed` to be more explicit than just `on_scheduler_message`

Reply:
I think that's a reasonable rename considering fjetter's comment